src/ceph-disk

```diff
@@ -1719,6 +1804,9 @@ def main_prepare(args):
 
         if args.zap_disk is not None:
             zap(args.data)
```

Comment:
Do we need to zap a device that has no partition table ? In which case do we have a device with no partition table that could cause a problem if not zapped ?